source/slang/slang-ir-autodiff-transpose.h

```diff
@@ -687,6 +711,36 @@ struct DiffTransposePass
         return tempRevVar;
     }
 
+    IRVar* getOrCreateInverseVar(IRInst* primalInst)
+    {
+        // No need to store inverse values for constants.
+        if (as<IRConstant>(primalInst))
+            return nullptr;
+
+        // Check if we have a var already.
+        if (inverseVarMap.ContainsKey(primalInst))
+            return inverseVarMap[primalInst];
+        
+        IRBuilder tempVarBuilder(autodiffContext->sharedBuilder);
```

Comment:
Use `autodiffContext->moduleInst` to initialize an `IRBuilder` from now on. `sharedBuilder` is removed from PR2657.